inference-engine/tests/functional/shared_test_classes/include/shared_test_classes/single_layer/convolution.hpp

```diff
@@ -43,6 +44,7 @@ class ConvolutionLayerTest : public testing::WithParamInterface<convLayerTestPar
 
 protected:
     void SetUp() override;
+    ngraph::PartialShape vec2partialshape(std::vector<std::vector<size_t>> inputShape);
```

Comment:
i think, we can move this method to the 'FuncTestUtils' because it can be reused

Reply:
The vec2partialshape is move to partial_shape_utils.hpp in functional_test_utils. Thanks for your comment.